tests/scripts/create-dev-cluster.sh

```diff
@@ -163,40 +170,30 @@ while getopts ":hrmfd:p:" opt; do
 	    minikube_profile_name="$OPTARG"
 	    ;;
 	\?)
-	    echo  "Invalid option: -$OPTARG" >&2
-	    show_usage
-	    exit 1
+	    invocation_error "Invalid option: -$OPTARG"
 	    ;;
 	:)
-	    echo "Option -$OPTARG requires an argument." >&2
-	    exit 1
+	    invocation_error "Option -$OPTARG requires an argument."
 	    ;;
     esac
 done
-
 echo "Using '$ROOK_EXAMPLES_DIR' as examples directory.."
-
 cd "$ROOK_EXAMPLES_DIR" || exit
 check_examples_dir
 init_vars "${minikube_profile_name:-rook}"
-
 if [ -z "$force_minikube" ]; then
     check_minikube_exists
 fi
-
 setup_minikube_env
 create_rook_cluster
 wait_for_rook_operator
 wait_for_ceph_cluster
-
 if [ "$enable_rook" = true ]; then
     enable_rook_orchestrator
 fi
-
 if [ "$enable_monitoring" = true ]; then
     enable_monitoring
 fi
-
 show_info "$enable_monitoring"
```

Comment:
It seems to me that all of these empty lines were probably intentional breaks to help breaking up the command flow. Is this an intentional and desired change?

Reply:
@BlaineEXE  wrote:
> It seems to me that all of these empty lines were probably intentional breaks to help breaking up the command flow. Is this an intentional and desired change?

Well, @rkachach  asked to remove some empty lines, but those were inside the definition of the new function. So I guess I removed a few too many. And when I noticed it, I decided  not  to undo the extra deletions, because I had recently been asked asked several times in PR reviews  to remove blank lines that I had added to stucture the code a bit for my own readability ...

Do you suggest to re-add those lines in the same commit?

Reply:
@obnoxxx @BlaineEXE is right, those lines break are intended for a better readability of the code. Without them IMHO it becomes really hard to read the code flow. Please, can you put them back? thanks.

Reply:
> @obnoxxx @BlaineEXE is right, those lines break are intended for a better readability of the code. Without them IMHO it becomes really hard to read the code flow. Please, can you put them back? thanks.

@rkachach  done. I hope I caught them all :-)